src/js/main.js

```diff
@@ -1,6 +1,13 @@
-import {animatedScrollTo} from './es6-scroll-to.js';
+// using jQuery
 
-document.querySelector('.title').innerHTML = 'Great success!'
-document.querySelector('.text').innerHTML = 'Javascript is working!'
+$('.contactUs').hide()
```

Comment:
This could lead to ugly flashes upon pageload if the user has slow internet connection. Just set `.contactUs` to `display:none` by default